### PR TITLE
fix(deps): update dependency `react-to-print` to `v3.0.1`

### PR DESCRIPTION
**Pull request by 0xTxbi**
# Description
This PR introduces changes that updates the `react-to-print` dependency to `v3.0.1`

# Issues
Refs #14771

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Review by julien51 — APPROVED** [review]

File: unlock-app/src/components/interface/Receipts/elements/ReceiptBox.tsx [hunk 1]
```diff
@@ -171,20 +171,30 @@ export const ReceiptBox = ({ lockAddress, hash, network }: ReceiptBoxProps) => {
                 {vatRatePercentage > 0 && (
                   <>
                     <Detail label="Subtotal" inline>
-                      {`${subtotal.toFixed(2)} ${symbol}`}
+                      {isPriceLoading
+                        ? 'Loading...'
```
Comment (julien51) [inline unlock-app/src/components/interface/Receipts/elements/ReceiptBox.tsx:175]: Can you use a `<Placeholder>` loader like we have?

File: unlock-app/src/components/interface/Receipts/elements/ReceiptBox.tsx [hunk 2]
```diff
@@ -171,20 +171,30 @@ export const ReceiptBox = ({ lockAddress, hash, network }: ReceiptBoxProps) => {
...
+                        : `${subtotal.toFixed(2)} ${symbol}`}
                     </Detail>
                     <Detail label={`VAT (${vatRatePercentage}%)`} inline>
-                      {vatTotalInAmount.toFixed(2)} {symbol}
+                      {isPriceLoading
+                        ? 'Loading...'
```
Comment (julien51) [inline unlock-app/src/components/interface/Receipts/elements/ReceiptBox.tsx:180]: same here

File: unlock-app/src/components/interface/Receipts/elements/ReceiptBox.tsx [hunk 3]
```diff
@@ -171,20 +171,30 @@ export const ReceiptBox = ({ lockAddress, hash, network }: ReceiptBoxProps) => {
...
                 )}
                 <Detail label="TOTAL" labelSize="medium" inline>
-                  <PriceFormatter
-                    price={(
-                      multiplier * parseFloat(receiptPrice?.total)
-                    ).toString()}
-                  />{' '}
-                  {symbol}
+                  {isPriceLoading ? (
+                    'Loading...'
```
Comment (julien51) [inline unlock-app/src/components/interface/Receipts/elements/ReceiptBox.tsx:187]: same :)

**Review by julien51 — CHANGES_REQUESTED** [review]
small nit!

**Comment by 0xTxbi** [comment]
> small nit!

good catch! all fixed!

**Review by julien51 — APPROVED** [review]

**Pull request merged**